3 - Assignment/240409 - Rust Basic + Roguelike Game, Week 2/geometry/src/lib.rs

```diff
@@ -1,14 +1,23 @@
+
+
 // Calculate the magnitude of a vector by summing the squares of its coordinates
 // and taking the square root. Use the `sqrt()` method to calculate the square
 // root, like `v.sqrt()`.
-fn magnitude(/* ... */) -> f64 {
-    todo!()
+fn magnitude(vec:&[f64;3]) -> f64 {
```

Comment:
`cargo fmt`를 통해 코드 스타일을 맞춰주세요.

```suggestion
fn magnitude(vec: &[f64; 3]) -> f64 {
```